server/simulation/missionProgress.js

```diff
@@ -18,13 +18,7 @@ module.exports = [
       return elapsedSeconds > 2;
     },
   },
-  {
```

Comment:
The new changes from `master` brought in a new status that also needs to be handled.